flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolUtils.java

```diff
@@ -55,6 +59,16 @@ private SlotPoolUtils() {
         throw new UnsupportedOperationException("Cannot instantiate this class.");
     }
 
+    public static DeclarativeSlotPoolBridge createDeclarativeSlotPoolBridge() {
+        return new DeclarativeSlotPoolBridge(
```

Comment:
Maybe use `DeclarativeSlotPoolBridgeBuilder` to build it?

Reply:
This won't work because the builder already starts the slot pool bridge which this method does not do. But I think, that I can change the `DeclarativeSlotPoolBridgeBuilder` a bit to make this work.